pkgs/by-name/ru/rustic/package.nix

```diff
@@ -44,11 +44,14 @@ rustPlatform.buildRustPackage rec {
     changelog = "https://github.com/rustic-rs/rustic/blob/${src.rev}/CHANGELOG.md";
     description = "fast, encrypted, deduplicated backups powered by pure Rust";
     mainProgram = "rustic";
-    platforms = lib.platforms.linux ++ lib.platforms.darwin;
+    platforms = with lib.platforms; linux ++ darwin;
     license = [
       lib.licenses.mit
       lib.licenses.asl20
     ];
-    maintainers = [ lib.maintainers.nobbz ];
+    maintainers = with lib.maintainers; [
+      nobbz
+      pmw
+    ];
```

Comment:
I really hate `with`, but if you really have to introduce it, because its used everywhere in nixpkgs already, then please do it everywhere and apply it to `license` as well.

So either:

- remove the use of `with` or
- use it for `license` as well

Reply:
Oh, I didn't know people dislike `with`. Ok, I will address this.